### ISSUE TITLE
Problem with running npm/node build due to unsatisfied version

### ISSUE THREAD

**Issue by marcusholl** [SAP/cloud-mta-build-tool#752]
When running a npm / node build which refers to `@sap/hdi-deploy: „3.10.0“` as only direct dependency we get:

```
npm WARN notsup Unsupported engine for @sap/hdi-deploy@3.10.0: wanted: {"node":">=6.9.1 <=8.15.0"} (current: {"node":"12.13.1","npm":"6.12.1"})
npm WARN notsup Not compatible with your version of node/npm: @sap/hdi-deploy@3.10.0 
```

The node and npm version comes from version `1.0.4` of `devxci/mbtci`.

Is there any other image available which provides node / npm with an approriate version. What should a user do who encouters a problem with an not suitable tool version?

**Comment by tal-sapan** [comment]
Hi,

Please try to update the version of `@sap/hdi-deploy`. The newest version should support node 10.

Best regards,
Tal

**Comment by marcusholl** [comment]
Thanks for your response. Issue can be solved by switching to the latest version `3.11.11`.
